gigboard-client/src/Models/Auth.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+class Auth {
+  
+  static register = ( userData ) => {
+    return axios.post('http://localhost:3001/api/v1/auth/register', userData );
```

Comment:
This looks really good, however I think in the backend we're going to use 'http://localhost:3001/register' but I'd like everybody's input. The reason Adonis put api/v1/auth was to keep it consistent with his movies route on the backend which was api/v1/movies. So personally I'd like to have it as just register here but we'll see what everyone else says :D